make_queue/views/reservation/reservation.py

```diff
@@ -29,10 +30,23 @@ def get_error_message(self, form, reservation):
         """
         if not reservation.is_within_allowed_period_for_reservation() and not (
                 reservation.special or reservation.event):
-            return f"Reservasjoner kan bare lages {reservation.reservation_future_limit_days} dager frem i tid"
+            """Translation: Reservasjoner kan bare lages {:} dager frem i tid"""
+            return _("Reservations can only be made {:} days ahead of time".format(
+                reservation.reservation_future_limit_days));
         if self.request.user.has_perm("make_queue.can_create_event_reservation") and form.cleaned_data["event"]:
-            return "Tidspunktet eller eventen, er ikke lengre tilgjengelig"
-        return "Tidspunktet er ikke lengre tilgjengelig"
+            """Translation: Tidspunktet eller eventen, er ikke lengre tilgjengelig"""
+            return _("The time slot or event, is no longer available")
+        if not reservation.quota_can_make_reservation():
+            """Translation: Reservasjonen går over kvoten"""
+            return _("The reservation exceeds the quota")
```

Comment:
A bit pedantic, but wouldn't it be better with `"The reservation exceeds your quota"`?

Reply:
Yeah, that sounds better